test/tools/ossfuzz/const_opt_ossfuzz.cpp

```diff
@@ -20,6 +20,9 @@
 
 using namespace std;
 
+// Prototype.
+extern "C" int LLVMFuzzerTestOneInput(uint8_t const* _data, size_t _size);
```

Comment:
@bshastry isn't there a header we could use for these prototypes?

Reply:
I think I just followed advice here: https://github.com/llvm-mirror/compiler-rt/blob/69445f095c22aac2388f939bedebf224a6efcdaf/lib/fuzzer/FuzzerInterface.h#L11-L14

I guess we could do `include <src/libfuzzer/FuzzerInterface.h>` since we include another interface header in this test harness https://github.com/ethereum/solidity/blob/4861c9b8d1cab36f84430cd6526d611f154db42a/test/tools/ossfuzz/abiV2ProtoFuzzer.cpp#L22

Reply:
Resolution: while it would be nice to use this header, unfortunately many of the fuzz targets are built on the regular CI run too, where it is not available. Someone should clean up the fuzz builds and only do them on `b_ossfuzz`.

Reply:
Let's track this separately #10573.